azure-pipelines.yml

```diff
@@ -0,0 +1,45 @@
+# Pipeline for the TeamUp's webApp
+
+trigger:
+  # batch runs this pipeline one after the other (useful when there are many contributors)
+  batch: true
+  branches:
+    include:
+      - master
+      - develop
+
+pr:
+  branches:
+    include:
+      - master
+      - develop
+  drafts: false
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '14.17.x' 
+  displayName: 'Installi Node 14.17.x'
+
+# installing dependencies (react, axios, etc)
```

Comment:
delete all comments please, let's keep using meaningful names for all tasks

Reply:
done